app/src/main/java/com/appunite/loudius/network/datasource/UserDataSource.kt

```diff
@@ -28,11 +27,7 @@ interface UserDataSource {
 
 @Singleton
 class UserDataSourceImpl @Inject constructor(
```

Comment:
please do not get mad at me :P but once again this example shows me that UserDataSource is not really helpfull, it is just a wrapper not adding anything new but creating unncessary jumping between classes ;p

Reply:
Honestly, I strongly agree in that case 😅